src/utils/filterUsersByRole.ts

```diff
@@ -0,0 +1,19 @@
+import { UserArray } from "../typeDefinitions/filterUsersByRole";
+
+export function filterUserByRoles(userArray: UserArray[], roleId: string) {
+  let filteredUser: string[] | [] = [];
+
+  if (userArray?.length > 0 && Array.isArray(userArray)) {
+    filteredUser = userArray?.reduce((acc: string[], current: UserArray) => {
+      // role present in user ?
```

Comment:
i dont think we need these comments here, what are your thoughts?

Reply:
removing it